lib/screens/IconExplanationsScreen.dart

```diff
@@ -0,0 +1,182 @@
+
+import 'package:flutter/material.dart';
+
+class IconExplanationsScreen extends StatefulWidget {
+  @override
+  createState() => _IconExplanationsScreenState();
+}
+
+class _IconExplanationsScreenState extends State<IconExplanationsScreen> {
+  bool _isLoading = false;
+
+  @override
+  Widget build(BuildContext context) {
+
+    Widget _body = Center(
+      child: Card(
+        color: Color.fromARGB(255, 224, 224, 224),
+        child: Container(
+          width: 400,
```

Comment:
There's a small overflow for the "School visit..." text. We can make the Container slightly wider or make the text wrap to avoid this issue. We can also leave it for now and fix it in #13.

![screenshot](https://user-images.githubusercontent.com/24862276/58969111-77c9f580-87b7-11e9-9a8d-f1fc720e515f.png)